<launch>
    <arg name="name_space" default="whole_body_controller/base"/>
    <rosparam command="load" ns="iai_donbot" file="$(find base_controller)/config/base_controller.yaml" />
    <node pkg="base_controller" type="base_controller.py" name="base_controller" output="screen">
        <remap from="~cmd_vel" to="/omnidrive/cmd_vel_twist"/>
        <param name="name_space" value="$(arg name_space)"/>
    </node>
</launch>
